libs/payments/ui/src/lib/nestapp/app.ts

```diff
@@ -20,7 +21,9 @@ class AppSingleton {
 
   async initialize() {
     if (!this.app) {
-      this.app = await NestFactory.createApplicationContext(AppModule);
+      this.app = await NestFactory.createApplicationContext(AppModule, {
+        logger: new ConsoleLogger(),
```

Comment:
Could this be be changed to use the `logger` from `@fxa/shared/log`? It's the same `logger` currently defined in the `nestapp/app.module.ts` and used by other NestJs services in the monorepo.

Reply:
Done!